### PR TITLE
PMM-4561 content in logs.zip grafana.log

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

File: ansible/pmm2/post-build-actions.yml [hunk 1]
```diff
@@ -67,3 +68,11 @@
         group: pmm
         mode: '0775'
 
+    - name: Create grafana.log file    | Create 'srv/logs/grafana.log' file
+      file:
+        path: /srv/logs/grafana.log
```
Comment (hors) [inline ansible/pmm2/post-build-actions.yml]: grafana.startup.log ?

File: ansible/pmm2/post-build-actions.yml [hunk 2]
```diff
@@ -37,6 +37,7 @@
         - /srv/logs/cron.log
         - /srv/logs/dashboard-upgrade.log
         - /srv/logs/grafana.log
+        - /srv/logs/grafana.startup.log
```
Comment (AlekSi) [inline ansible/pmm2/post-build-actions.yml]: :+1:

File: ansible/pmm2/post-build-actions.yml [hunk 3]
```diff
@@ -67,3 +68,11 @@
         group: pmm
         mode: '0775'
 
+    - name: Create grafana.log file    | Create 'srv/logs/grafana.log' file
```
Comment (hors) [inline ansible/pmm2/post-build-actions.yml]: /srv/logs/grafana.log

Comment (AlekSi) [inline ansible/pmm2/post-build-actions.yml]: Not sure about this one. I would say that everything that can log to stdout/stderr should do so, and let supervisord capture those streams and rotate them. We want to move away from processes logging to files themselves, and from logrotate (and crond).

**Pull request closed without merging**